packages/hyperion-autologging/src/AutoLogging.ts

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
this is actually the former `index.ts` and I used `git mv` to rename the file, not sure why GH shows it this way.